.dockerignore

```diff
@@ -0,0 +1,7 @@
+*
+!updateHostsFile.py
+!data
+!alternates
+!extensions
+!readmeData.json
+!requirements.txt
```

Comment:
@rhtenhove can you briefly explain why these files belong in `.dockerignore`?  This seems counter intuitive to me but, then again, what do I know?

Reply:
It's actually a reverse. So we dockerignore everything with `*`, except (`!`) for the files and folders we actually need. Allows for a much cleaner ignorefile, prevents adding unneccesary new stuff, and improves caching as not every little update busts the cache.